### DISCUSSION TITLE
Classic Shell upgrade to Open Shell?

### DISCUSSION THREAD

**Discussion by hspindel** [Open-Shell/Open-Shell-Menu#1716] [Q&A] [answered]
I have used Classic Shell successfully for years.

If I install Open Shell, will it automatically replace Classic Shell?  Or do I need to uninstall Classic Shell?  If there is no automatic upgrade, how are Classic Shell settings migrated to Open Shell?

Using Win 10.

**Comment by ge0rdi** [comment, marked as answer]
`Open-Shell` can be installed over `Classic Shell`.
It should migrate previous settings just well.

**Reply by hspindel** [reply]
Thank you.
